src/items/type-aliases.md

```diff
@@ -34,11 +34,15 @@ let _ = TypeAlias(5); // Doesn't work
 A type alias without the [_Type_] specification may only appear as an
 [associated type] in a [trait].
 
+A type alias with the [_Type_] specification may only appear as an
+[associated type] in a [trait impl].
+
 A type alias with [_TypeParamBounds_] may only specified when used as
 an [associated type] in a [trait].
 
-A type alias with where clauses after the equals sign may only appear as an
-[associated type] in a [trait] or a [trait impl].
+Where clauses before the equals sign on a type alias in a [trait impl] (like 
```

Comment:
You have the same example of code in both parenthesis. Also, there's no reason to use parentheis; just use commas.

Reply:
I fixed the second example. Parentheses make more sense to me, since they are somewhat of an aside and aren't needed to understand the sentence, only add additional clarification.